src/resources/ResourcePhrases.ts

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable max-len */
```

Comment:
I thought I'd have the strings all on one line (no breaks). Thoughts?

Reply:
👍